extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -0,0 +1,510 @@
+						'width': `1100px`,
+						'height': '300px',
+						'background-size': '100%',
+					}
+				});
+				header.addItem(tasksContainer, {
+					CSSStyles: {
+						'width': `${maxWidth}px`,
+						'height': '150px',
+					}
+				});
+
+				header.addItem(await this.createFooter(view), {
+					CSSStyles: {
+						'margin-top': '20px'
+					}
+				});
+
+				const mainContainer = view.modelBuilder.flexContainer()
+					.withLayout({
+						flexFlow: 'column',
+						width: '100%',
+						height: '100%',
+						position: 'absolute'
+					}).component();
+				mainContainer.addItem(container, {
+					CSSStyles: { 'padding-top': '25px', 'padding-left': '5px' }
+				});
+				await view.initializeModel(mainContainer);
+
+				this.refreshMigrations();
+				resolve();
+			});
+		});
+	}
+
+	private createHeader(view: azdata.ModelView): azdata.FlexContainer {
+		const header = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			width: maxWidth,
+		}).component();
+		const titleComponent = view.modelBuilder.text().withProperties({
+			value: loc.DASHBOARD_TITLE,
+			CSSStyles: {
+				'font-size': '36px',
+				'font-family': 'Segoe UI'
+			}
+		}).component();
+		const descComponent = view.modelBuilder.text().withProperties({
+			value: loc.DASHBOARD_DESCRIPTION,
+			CSSStyles: {
+				'font-size': '12px',
+			}
+		}).component();
+		header.addItems([titleComponent, descComponent], {
+			CSSStyles: {
+				'width': `${maxWidth}px`,
+				'padding-left': '20px'
+			}
+		});
+
+		return header;
+	}
+
+	private asAbsolutePath(filePath: string): string {
+		return path.join(this._root || '', filePath);
+	}
+
+	private async createTasks(view: azdata.ModelView): Promise<azdata.Component> {
+		const tasksContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'row',
+			width: '100%',
+			height: '50px',
+		}).component();
+
+		const migrateButtonMetadata: IActionMetadata = {
+			title: loc.DASHBOARD_MIGRATE_TASK_BUTTON_TITLE,
+			description: loc.DASHBOARD_MIGRATE_TASK_BUTTON_DESCRIPTION,
+			iconPath: {
+				dark: this.asAbsolutePath('images/migration.svg'),
+				light: this.asAbsolutePath('images/migration.svg'),
+			},
+			command: 'sqlmigration.start'
+		};
+		const migrateButton = this.createTaskButton(view, migrateButtonMetadata);
+
+		const preRequisiteListHTMLTemplate =
+			`
+			<span style="font-size: 14px;">Things you need before starting migration:</span><br>
+			<ul style="padding:0 0 0 10px;font-size:12px;">
+			<li>Azure account details</li>
+			<li>Azure SQL Managed Instance or SQL Server on Azure Virtual Machine</li>
+			<li>Backup location details</li>
+			</ul>
+			`;
+
+		const preRequisiteListElement = view.modelBuilder.dom().withProps({
+			html: preRequisiteListHTMLTemplate
+		}).component();
+
+		const preRequisiteLearnMoreLink = view.modelBuilder.hyperlink().withProps({
+			label: loc.LEARN_MORE,
+			url: '',
```

Comment:
I assume this will be added later? Make sure you're tracking stuff like this somewhere - it's easy to miss stuff like this